src/transformers/trainer_tf.py

```diff
@@ -273,45 +284,44 @@ def train(self) -> None:
         logger.info("  Num Epochs = %d", epochs)
         logger.info("  Total optimization steps = %d", self.train_steps)
 
+        self.global_step = 0
```

Comment:
This variable seems a repetition with the `iterations` variable can be removed.

Reply:
This is to make it compatible with the functions moved from `Trainer` to `TFTrainer`.
Could `iterations` be removed or does it capture something more complex (such as reloading from checkpoints…).

Reply:
It captures the step directly from the checkpoint if any so, it is a real duplicate of global.